### PR TITLE
Lensa.homepage

### PR DESCRIPTION
**Pull request by lensabillion**

### CODE REVIEW

File: Frontend/pages/index.vue [hunk 1]
```diff
@@ -59,7 +59,7 @@
               Who We Are
             </h1>
             <p class="text-body">
-              Emre Varol, a former competitive programmer and <b>Google Research, Palantir</b>, Liftoff software/ML engineer launched A2SV in Ethiopia through a partnership with Addis Ababa University in October 2019. In July 2020, <b>Burak Saglam</b>, a former <b>International Math Olympiad medalist</b>, and <b>Erden Oktay</b>(MSc Tufts University), senior software engineer, joined.
+             Emre Varol, a former competitive programmer and Google Research, Palantir, Liftoff software/ML engineer launched A2SV in Ethiopia through a partnership with Addis Ababa University in October 2019. Currently, A2SV has a team of more than 40 members participating in its education and project development efforts.
```
Comment (LydiaGA) [inline Frontend/pages/index.vue:62]: We forgot about these <b> tags. I think we have to add them

**Comment by LydiaGA** [comment]
Everything else looks good.

File: Frontend/pages/index.vue [hunk 2]
```diff
@@ -70,8 +70,7 @@
             Our Belief
           </h1>
           <p class="text-body text-center">
-            The world is yet to see that talent is everywhere, but the opportunity is not.
-            That is why we want to provide quality education in underdeveloped areas.
+            The world is yet to see that talent is everywhere, but opportunity is not. That is why we want to provide quality education in underdeveloped areas.
```
Comment (YiseBoge) [inline Frontend/pages/index.vue:73]: If possible, better to lint the file to make it more readable

**Review by YiseBoge — COMMENTED** [review]
Shy of some linting issues, it looks good to me

**Pull request merged**